### PR TITLE
Fixed the nbody bruteforce example

### PR DESCRIPTION
**Pull request by sbalint98**
This PR should fix the nbody example. Now it compiles without warnings.

### CODE REVIEW

**Comment by illuhad** [comment]
Thanks, great to have this example finally fixed :)

File: examples/bruteforce_nbody/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 add_executable(bruteforce_nbody bruteforce_nbody.cpp)
-target_link_libraries(bruteforce_nbody hipSYCL)
+target_link_libraries(bruteforce_nbody)
```
Comment (illuhad) [inline examples/bruteforce_nbody/CMakeLists.txt]: You can remove this line entirely - in the current formulation, it just says, for target `bruteforce_nbody` link with libraries in `<empty set>`. If we are at it, maybe we should think about doing a proper cmake integration using `find_package(hipSYCL)` and `add_sycl_to_target()`. I can give you some examples if you are interested.

Comment (sbalint98) [reply examples/bruteforce_nbody/CMakeLists.txt]: Thank you, I'd like to do a proper CMake integration, so can you give me some examples?

File: examples/CMakeLists.txt [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
-  message(SEND_ERROR "hipSYCL syclcc compiler not found, exiting.")
-  return()
-endif()
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/../cmake)
 
-set(CMAKE_CXX_COMPILER ${SYCLCC})
-project(hipsycl-examples)
+find_package(hipSYCL CONFIG REQUIRED)
 
+set(CMAKE_CXX_COMPILER ${SYCLCC})
```
Comment (illuhad) [inline examples/CMakeLists.txt]: This should not be necessary. `add_sycl_to_target` should take care of this.

Comment (sbalint98) [reply examples/CMakeLists.txt]: got it!

**Comment by illuhad** [comment]
Thank you!

**Pull request merged**